src/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 add_executable (aligner
         main.cpp
-        SingleAligner.cpp
-        SingleAligner.hpp
-)
+        SingleAligner.cpp SingleAligner.hpp parsers/FileParserSmiles.h
```

Comment:
header files can be removed (please also remove the SingleAligner.hpp :) ) .
And let us use .hpp file instead of .h